src/sdk/model/company/company.ts

```diff
@@ -420,6 +432,43 @@ export class Company extends Entity {
     });
   }
 
+  /**
+   * Gets all the O365 Organizations in the company location.
+   * @param {string} locationId
+   * @return {Promise<Array<O365Organization>>}
+   */
+  /* istanbul ignore next: autogenerated */
+  async getO365Organizations(locationId: string): Promise<Array<O365Organization>> {
+    return Iland.getHttp().get(`/companies/${this.uuid}/location/${locationId}/o365-organizations`)
+        .then((response) => {
+          const json = response.data.data as Array<O365OrganizationJson>;
+          return json.map((it) => new O365Organization(it));
+        });
+  }
+
+  /**
+   * Get O365 bill for Company
+   * @param locationId
+   * @param startYear
+   * @param startMonth
+   * @param endYear
+   * @param endMonth
+   */
+  async getO365billing(locationId: string, startYear?: number, startMonth?: number,
+                       endYear?: number, endMonth?: number): Promise<Array<O365Bill>> {
```

Comment:
Since some of the params are optional, should they have fallback or default values? ex. getSecurityGroupMemberOptions

Reply:
Great question! If no dates the API returns the last 6 months as default. I added a comment on the docs for this